app/api/posts/[identifier]/likes/route.ts

```diff
@@ -0,0 +1,42 @@
+import { auth } from '@/app/api/auth/[...nextauth]/auth';
+import config from '@/config';
+import { MumblePostService } from '@/services/mumble/MumblePost';
+import { NextRequest, NextResponse } from 'next/server';
+
+const dataSource = new MumblePostService(config.mumble.host);
+
+export const PUT = async (
```

Comment:
route.ts files in posts and users directories are replaced by server actions in this PR: https://github.com/ost-cas-fee-adv-23-24/nextjs-app-football-fanatics/pull/11